.CMake/alg_support.cmake

```diff
@@ -468,61 +468,133 @@ if(OQS_DIST_X86_64_BUILD OR (OQS_USE_AVX2_INSTRUCTIONS))
     cmake_dependent_option(OQS_ENABLE_SIG_sphincs_shake256_128f_robust_avx2 "" ON "OQS_ENABLE_SIG_sphincs_shake256_128f_robust" OFF)
 endif()
 
+if(CMAKE_SYSTEM_NAME MATCHES "Linux|Darwin")
+if(OQS_DIST_ARM64_V8_BUILD OR (OQS_USE_ARM_NEON_INSTRUCTIONS AND OQS_USE_SHA3_INSTRUCTIONS))
```

Comment:
This looks _very_ fishy: Should this rather read "OQS_USE_ARM_SHA3_INSTRUCTIONS"?

Reply:
Possibly fixed by https://github.com/open-quantum-safe/liboqs/pull/1421

Reply:
Thanks!